app.js

```diff
@@ -25,4 +25,164 @@ window.addEventListener('DOMContentLoaded', () => {
+    },
+    {
+      id: 3,
+      title: 'Multi-Post Stories Gain+Glory',
+      techStack: ['HTML', 'CSS', 'JavaScript', 'Ruby on Rails'],
+      screenshotMobile: '#',
+      screenshotDekstop: '#',
+      details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s",
+      source: 'https://github.com/ichsansandy/portfolio-microverse',
+      live: 'https://github.com/ichsansandy/portfolio-microverse',
+    },
+    {
+      id: 4,
+      title: 'Multi-Post Stories Gain+Glory',
+      techStack: ['HTML', 'CSS', 'JavaScript', 'Ruby on Rails'],
+      screenshotMobile: '#',
+      screenshotDekstop: '#',
+      details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s",
+      source: 'https://github.com/ichsansandy/portfolio-microverse',
+      live: 'https://github.com/ichsansandy/portfolio-microverse',
+    },
+    {
+      id: 5,
+      title: 'Multi-Post Stories Gain+Glory',
+      techStack: ['HTML', 'CSS', 'JavaScript', 'Ruby on Rails'],
+      screenshotMobile: '#',
+      screenshotDekstop: '#',
+      details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s",
+      source: 'https://github.com/ichsansandy/portfolio-microverse',
+      live: 'https://github.com/ichsansandy/portfolio-microverse',
+    },
+    {
+      id: 6,
+      title: 'Multi-Post Stories Gain+Glory',
+      techStack: ['HTML', 'CSS', 'JavaScript', 'Ruby on Rails'],
+      screenshotMobile: '#',
+      screenshotDekstop: '#',
+      details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s",
+      source: 'https://github.com/ichsansandy/portfolio-microverse',
+      live: 'https://github.com/ichsansandy/portfolio-microverse',
+    },
+  ];
+
+  const cards = () => {
+    const card = portfolioList.map(
+      (portfolio) => `<div class="card">
+      <div class="card-image" style="--image:url('${portfolio.screenshotMobile}')"></div>
+      <div class="card-info">
+      <h4 class="card-title title-text center-text dark">${portfolio.title}</h4>
+      <ul class="tag-container">
+      <li class="tag flex-center">${portfolio.techStack[0]}</li>
+      <li class="tag flex-center">${portfolio.techStack[1]}</li>
+      <li class="tag flex-center">${portfolio.techStack[2]}</li>
+      <li class="tag flex-center">${portfolio.techStack[3]}</li>
+      </ul>
```

Comment:
[OPTIONAL]
- [ ] To make your PR more dynamic because not every project will have the same number of tech tasks, you may consider looping through this array and not hard coding the item as you just did